pages/detailed-example-workflow/Step_6_Taxa_and_Gene_Selection.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Taxa and Gene Selection
+parent: Detailed Example Workflow
+nav_order: 6
+permalink: /detailed-example-workflow/step6-Taxa-and-Gene_Selection
+---
+
+# Select Taxa and Orthologs to be Included in a Phylogenomic Dataset
+
+To select taxa and orthologs to be included in the final phylogenomic dataset users can run the scripts
+`select_taxa.py` and `select_orthologs.py` respectively. Each will generate a .tsv file (contents explained below) that will serve to select either taxa or orthologs to include and both .tsv files are used as input for the script `prep_final_dataset.py` downstream.
```

Comment:
Not sure how much, if it all, it matters, but I've been deleting the new line if it's in the middle of a sentence